ca_cdk_constructs/eks/imports/io/external_secrets/generators/py.typed

```diff
@@ -0,0 +1 @@
+
```

Comment:
not sure why this additional blank line but ok...

Reply:
This file was from the `cdk8s import --output ca_cdk_constructs/eks/imports` command as well.